components/navbar.py

```diff
@@ -34,24 +34,30 @@ def my_navbar():
                         # Use the Col to contain A tag for LOGO
                         dbc.Col(
                             # Make the LOGO can navigate to other website
-                            html.A(
+                            dmc.Anchor(
                                 dbc.Col(
-                                    html.Img(
+                                    dmc.Image(
                                         src="assets/media/logo-placeholder.png",
-                                        height="80px",
+                                        style={  # Dynamic adjust size in small screen
+                                            "width": "min(12vw, 70px)",
+                                            "max-width": "70px",
+                                        },
                                     ),
                                 ),
                                 href=URLS.HOME.value,
                             ),
-                            className="mx-auto",
                             width="auto",
                         ),
                         # The title
                         dbc.Col(
-                            html.H1(
+                            dmc.Text(
                                 app_name,
-                                className="text-center mb-0",
-                                style={"color": "#0c2772", "white-space": "nowrap"},
+                                c="#0c2772",
+                                style={  # Dynamic adjust size in small screen
+                                    "font-size": "min(5vw, 32px)"
+                                },
+                                truncate=True,  # text in single line
```

Comment:
you should be careful about this truncate=True since it will mess up the navbar if the screen is very thin